tools/gpstest/SConstruct

```diff
@@ -0,0 +1,8 @@
+env = Environment(
+        CFLAGS = ["-O2"],
+        CC='gcc',
+    )
+
+src = ["src/limegps.c", "src/gpssim.c"]
+libs = ["LimeSuite", "pthread", "m"]
+env.Program("bin/LimeGPS", src, LIBS=libs, LIBPATH=['lib'])
```

Comment:
can't we just run `make`?

Reply:
sure, but we use SConscript everywhere else^^

Reply:
that's only for our stuff. for example, what happens if gpssim.c gets renamed to gps_sim.c? `make` will still work, but this scons stuff will have to be updated